src/style.css

```diff
@@ -1 +1,75 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 12px;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  --main-color: #00acdc;
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  line-height: 14px;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+  display: flex;
+  margin: auto 0;
+}
+
+.nav {
+  text-transform: uppercase;
+}
+
+.nav_list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.nav_item + .nav_item {
+  margin: 0 0 0 20px;
+}
```

Comment:
Don't increase specificity, use pseudoclasses here (:not and child)
And use specific margin property, like margin-left.